src/components/buttons/Buttons.jsx

```diff
@@ -0,0 +1,14 @@
+import { MARK_ICON, PEN_ICON, BIN_ICON } from "src/utils/constants/constants";
+import { createButton } from "src/utils/helpers/createButton";
+
+const Buttons = () => {
+  return (
+    <div>
```

Comment:
```suggestion
    <>
```

replace div with <></> in all files where it isn't necessary

Reply:
ditto